doc/examples/cloud-config-rsyslog.txt

```diff
@@ -5,34 +5,34 @@
 ## Example:
 #cloud-config
 rsyslog:
- remotes:
-  # udp to host 'maas.mydomain' port 514
-  maashost: maas.mydomain
-  # udp to ipv4 host on port 514
-  maas: "@[10.5.1.56]:514"
-  # tcp to host ipv6 host on port 555
-  maasipv6: "*.* @@[FE80::0202:B3FF:FE1E:8329]:555"
- configs:
-   - "*.* @@192.158.1.1"
-   - content: "*.*   @@192.0.2.1:10514"
-     filename: 01-example.conf
-   - content: |
-      *.*   @@syslogd.example.com
- config_dir: /etc/rsyslog.d
- config_filename: 20-cloud-config.conf
- service_reload_command: [your, syslog, reload, command]
+  remotes:
+    # udp to host 'maas.mydomain' port 514
+    maashost: maas.mydomain
+    # udp to ipv4 host on port 514
+    maas: "@[10.5.1.56]:514"
+    # tcp to host ipv6 host on port 555
+    maasipv6: "*.* @@[FE80::0202:B3FF:FE1E:8329]:555"
+  configs:
+    - "*.* @@192.158.1.1"
+    - content: "*.*   @@192.0.2.1:10514"
+      filename: 01-example.conf
+    - content: |
+       *.*   @@syslogd.example.com
+  config_dir: /etc/rsyslog.d
+  config_filename: 20-cloud-config.conf
+  service_reload_command: [your, syslog, reload, command]
 
 ## Additionally the following legacy format is supported
 ## it is converted into the format above before use.
 ##  rsyslog_filename -> rsyslog/config_filename
 ##  rsyslog_dir -> rsyslog/config_dir
 ##  rsyslog -> rsyslog/configs
 # rsyslog:
-#  - "*.* @@192.158.1.1"
-#  - content: "*.*   @@192.0.2.1:10514"
-#    filename: 01-example.conf
-#  - content: |
-#     *.*   @@syslogd.example.com
+#   - "*.* @@192.158.1.1"
+#   - content: "*.*   @@192.0.2.1:10514"
+#     filename: 01-example.conf
+#   - content: |
+#      *.*   @@syslogd.example.com
```

Comment:
This indentation is not a multiple of 2, and is inconsistent with the indentation [below](https://github.com/canonical/cloud-init/pull/323/files#diff-55f586d63e2327a86b885884e6945d65R16). Here we have:

```
#   - content: |
#      *.*   @@syslogd.example.com
```

below we have

```yaml
- content: |
    # My new /etc/sysconfig/samba file
```

(I do like the double indent of `|` content, I think it makes it easier to read.)

Reply:
Oh, you're right! fixup 7fb59577